libs/community/langchain_community/chains/__init__.py

```diff
@@ -0,0 +1,27 @@
+"""
+Chains module for langchain_community
+
+This module contains the community chains.
+"""
+
+import importlib
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING:
+    from langchain_community.chains.pebblo_retrieval.base import PebbloRetrievalQA
+
+__all__ = ["PebbloRetrievalQA"]
```

Comment:
It should use a hard-coded list of string literals otherwise some dev tools won't be able to pick it up